benchmarks/benchmark_quantize_triton.py

```
@@ -0,0 +1,258 @@
+
+def benchmark_cuda(func, x, scale, zero_point, q_min, q_max, args, name, warmup=False):
+    """Benchmark a quantization function on CUDA."""
+    x = x.clone()
+    if warmup:
+        print(f"  Warming up {name}...")
+        for _ in range(10):
+            _ = func(x, scale, zero_point, q_min, q_max, args)
+        torch.cuda.empty_cache()
+        gc.collect()
+        torch.cuda.synchronize()
+        print(f"  Warmup complete, starting benchmark...")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)